src/promptflow/promptflow/contracts/run_info.py

```diff
@@ -20,16 +22,45 @@ class Status(Enum):
 
     @staticmethod
     def is_terminated(status):
+        """Check if a given status is terminated.
+
+        :param status: The status to be checked
+        :return: True if the status is terminated, False otherwise
+        :rtype: bool
+        """
         if isinstance(status, Status):
             status = status.value
         return status in {s.value for s in {Status.Completed, Status.Failed, Status.Bypassed, Status.Canceled}}
 
 
 @dataclass
 class RunInfo:
-    node: str  # Node name
-    flow_run_id: str  # This is equal to root_run_id
-    run_id: str  # flow_run_id:step_run_id
+    """A dataclass representing the run information.
+
+    :param node: Node name
```

Comment:
you do not have type spec for each arguments? 

```
:type uuids: list, optional
```

Reply:
I thought it is optional because the example in the excel did not have type spec in the docstring for class definition, and I have added type spec now. <img width="750" alt="1694684244546" src="https://github.com/microsoft/promptflow/assets/39176492/8f209366-3556-4e47-9508-ca4820d0a036">